### PR TITLE
Release notes for 5.19

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Release notes for 5.19.0.

## Motivation and Context
Support 5.19.0 release.

### CODE REVIEW

File: content/sensu-go/5.19/release-notes.md [hunk 1]
```diff
@@ -55,6 +56,33 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+**Release summary placeholder**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.19.0.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][122]) In the [web UI][124], you can now [save, recall, and delete filtered searches][123].
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- **NEED TO REVISIT** ([Commercial feature][122]) Added `contains` as an operator for API response, sensuctl, and dashboard filtering selectors.
+- **NEED TO REVISIT** ([Commercial feature][122]) Added agent and sensuctl builds for Linux architectures: `mips`, `mipsle`, `mips64`, and `mips64le` (softfloat and hardfloat).
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
- ([Commercial feature][122]) Added agent and sensuctl builds for Linux architectures: `mips`, `mipsle`, `mips64`, and `mips64le` (hard float and soft float).
```

File: content/sensu-go/5.19/release-notes.md [hunk 2]
```diff
@@ -1111,3 +1139,7 @@ To get started with Sensu Go:
 [119]: /sensu-go/5.18/api/overview/#response-filtering
 [120]: /sensu-go/5.18/api/auth/#the-authtest-api-endpoint
 [121]: /sensu-go/5.18/getting-started/enterprise/
+[122]: /sensu-go/5.19/getting-started/enterprise/
+[123]: /sensu-go/5.19/dashboard/filtering/#save-a-filtered-search
+[124]: /sensu-go/5.19/dashboard/overview/
+[125]: /sensu-go/5.19/api/health/
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md:1153]:
```suggestion
[125]: /sensu-go/5.19/api/health/
[126]: /sensu-go/5.19/api/overview/#response-filtering
[127]: /sensu-go/5.19/sensuctl/reference/#response-filtering
[128]: /sensu-go/5.19/dashboard/filtering
```

File: content/sensu-go/5.19/release-notes.md [hunk 3]
```diff
@@ -55,6 +56,33 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**Release summary placeholder**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.19.0.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][122]) In the [web UI][124], you can now [save, recall, and delete filtered searches][123].
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- **NEED TO REVISIT** ([Commercial feature][122]) Added `contains` as an operator for API response, sensuctl, and dashboard filtering selectors.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
- ([Commercial feature][122]) Added the `matches` substring matching operator for [API response][126], [sensuctl][127], and [dashboard][128] filtering selectors.
```

File: content/sensu-go/5.19/release-notes.md (hunk 2 above)
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md:1153]:
```suggestion
[125]: /sensu-go/5.19/api/health/
[126]: /sensu-go/5.19/api/overview/#response-filtering
[127]: /sensu-go/5.19/sensuctl/reference/#response-filtering
[128]: /sensu-go/5.19/dashboard/filtering
```

File: content/sensu-go/5.19/release-notes.md [hunk 4]
```diff
@@ -55,6 +56,33 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.19.0 release notes
+
+**March 25, 2020** &mdash; The latest release of Sensu Go, version 5.19.0, is now available for download.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
**March 26, 2020** &mdash; The latest release of Sensu Go, version 5.19.0, is now available for download.
```

File: content/sensu-go/5.19/release-notes.md [hunk 5]
```diff
@@ -55,6 +56,33 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.19.0.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][122]) In the [web UI][124], you can now [save, recall, and delete filtered searches][123].
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- ([Commercial feature][122]) Added the `matches` substring matching operator for [API response][126], [sensuctl][127], and [dashboard][128] filtering selectors.
+- ([Commercial feature][122]) Added agent and sensuctl builds for Linux architectures: `mips`, `mipsle`, `mips64`, and `mips64le` (hard float and soft float).
+- **NEED TO REVISIT**  The `sensu.io/managed_by` label is now automatically applied to resources created via `sensuctl create`.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
- ([Commercial feature][122]) Sensu now automatically applies the `sensu.io/managed_by` label to resources created via `sensuctl create` for use in the [`sensuctl prune` alpha feature][129].
```

File: content/sensu-go/5.19/release-notes.md [hunk 6]
```diff
@@ -1111,3 +1139,13 @@ To get started with Sensu Go:
...
+[122]: /sensu-go/5.19/getting-started/enterprise/
+[123]: /sensu-go/5.19/dashboard/filtering/#save-a-filtered-search
+[124]: /sensu-go/5.19/dashboard/overview/
+[125]: /sensu-go/5.19/api/health/
+[126]: /sensu-go/5.19/api/overview/#response-filtering
+[127]: /sensu-go/5.19/sensuctl/reference/#response-filtering
+[128]: /sensu-go/5.19/dashboard/filtering
+[126]: /sensu-go/5.19/api/overview/#response-filtering
+[127]: /sensu-go/5.19/sensuctl/reference/#response-filtering
+[128]: /sensu-go/5.19/dashboard/filtering
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md:1159]:
```suggestion
[128]: /sensu-go/5.19/dashboard/filtering
[129]: /sensu-go/5.19/sensuctl/reference/#prune-resources
```

File: content/sensu-go/5.19/release-notes.md [hunk 7]
```diff
@@ -55,6 +56,33 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- ([Commercial feature][122]) Added the `matches` substring matching operator for [API response][126], [sensuctl][127], and [dashboard][128] filtering selectors.
+- ([Commercial feature][122]) Added agent and sensuctl builds for Linux architectures: `mips`, `mipsle`, `mips64`, and `mips64le` (hard float and soft float).
+- ([Commercial feature][122]) Sensu now automatically applies the `sensu.io/managed_by` label to resources created via `sensuctl create` for use in the [`sensuctl prune` alpha feature][129].
+- Resource metadata now includes the `created_by` field, which Sensu automatically populates with the name of the user who created or last updated each resource.
+- The agent now discovers entity libc type, VM system, VM role, and cloud provider.
+- System type now includes the `float_type` field, which stores the float type the system is using (softfloat or hardfloat).
+- The Bonsai client now logs at the `debug` level rather than the `info` level.
+- The store can now create wrapped resources.
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md:85]:
```suggestion
- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu, and all Tessen transmissions are logged for complete transparency.
```

File: content/sensu-go/5.19/release-notes.md [hunk 8]
```diff
@@ -1111,3 +1139,14 @@ To get started with Sensu Go:
...
+[123]: /sensu-go/5.19/dashboard/filtering/#save-a-filtered-search
+[124]: /sensu-go/5.19/dashboard/overview/
+[125]: /sensu-go/5.19/api/health/
+[126]: /sensu-go/5.19/api/overview/#response-filtering
+[127]: /sensu-go/5.19/sensuctl/reference/#response-filtering
+[128]: /sensu-go/5.19/dashboard/filtering
+[126]: /sensu-go/5.19/api/overview/#response-filtering
+[127]: /sensu-go/5.19/sensuctl/reference/#response-filtering
+[128]: /sensu-go/5.19/dashboard/filtering
+[129]: /sensu-go/5.19/sensuctl/reference/#prune-resources
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md:1160]:
```suggestion
[129]: /sensu-go/5.19/sensuctl/reference/#prune-resources
[130]: /sensu-go/5.19/reference/tessen/
```

File: content/sensu-go/5.19/release-notes.md [hunk 9]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.19.0 release notes
+
+**March 26, 2020** &mdash; The latest release of Sensu Go, version 5.19.0, is now available for download.
+This release is packed with new features, improvements, and fixes, including our first alpha feature: sensuctl pruning to help keep your configuration in good order.
```
Comment (cwjohnston) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
This release is packed with new features, improvements, and fixes, including our first *feature preview* in a release: declarative configuration pruning to help keep your Sensu instance in sync with Infrastructure as Code workflows.
```

File: content/sensu-go/5.19/release-notes.md [hunk 10]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- Resource metadata now includes the `created_by` field, which Sensu automatically populates with the name of the user who created or last updated each resource.
+- The agent now discovers entity libc type, VM system, VM role, and cloud provider.
+- System type now includes the `float_type` field, which stores the float type the system is using (hard float or soft float).
+- The Bonsai client now logs at the `debug` level rather than the `info` level.
+- The store can now create wrapped resources.
+- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu, and all Tessen transmissions are logged for complete transparency.
```
Comment (cwjohnston) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu, and all Tessen transmissions are logged locally for complete transparency.
```

File: content/sensu-go/5.19/release-notes.md [hunk 11]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.19.0 release notes
+
+**March 26, 2020** &mdash; The latest release of Sensu Go, version 5.19.0, is now available for download.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
**March 30, 2020** &mdash; The latest release of Sensu Go, version 5.19.0, is now available for download.
```

File: content/sensu-go/5.19/release-notes.md [hunk 12]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- Resource metadata now includes the `created_by` field, which Sensu automatically populates with the name of the user who created or last updated each resource.
+- The agent now discovers entity libc type, VM system, VM role, and cloud provider.
+- System type now includes the `float_type` field, which stores the float type the system is using (hard float or soft float).
+- The Bonsai client now logs at the `debug` level rather than the `info` level.
+- The store can now create wrapped resources.
+- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu, and all Tessen transmissions are logged for complete transparency.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and logs numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu.
```

File: content/sensu-go/5.19/release-notes.md [hunk 13]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- Resource metadata now includes the `created_by` field, which Sensu automatically populates with the name of the user who created or last updated each resource.
+- The agent now discovers entity libc type, VM system, VM role, and cloud provider.
+- System type now includes the `float_type` field, which stores the float type the system is using (hard float or soft float).
+- The Bonsai client now logs at the `debug` level rather than the `info` level.
+- The store can now create wrapped resources.
+- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and logs numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu.
+
+**FIXES:**
```
Comment (echlebek) [inline content/sensu-go/5.19/release-notes.md:86]: We're missing one of the fixes:

- Fixed a bug where check history was incorrectly formed when using the postgres event store.

File: content/sensu-go/5.19/release-notes.md [hunk 14]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+- Fixed a bug where `event.Check.State` was not set for events passing through the pipeline or written to the event log.
+- Fixed a bug that allowed the agent to connect to a backend using a nonexistent namespace.
+- Fixed a bug that allowed `subscriptions` to be empty strings.
+- Corrected the HTTP status codes for unauthenticated and permission denied errors in the REST API.
+
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
- Fixed a bug where check history was incorrectly formed when using the PostgreSQL event store.
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
One suggestion here.

I think this release includes a number of visual changes in the web UI. Is that true? I thought we had an internal demo of new UI work but  I don’t see any mention here and I’m not sure how I would summarize those changes .

**Comment by cwjohnston** [comment]
Idk if “feature preview” feels better than “alpha” for users... either way `alpha` will be part of the API route so maybe it’s irrelevant   @calebhaliey wdyt?

File: content/sensu-go/5.19/release-notes.md [hunk 15]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+**IMPROVEMENTS:**
+
+- ([Commercial feature][122]) The [health endpoint][125] now includes PostgreSQL health information.
+- Resource metadata now includes the `created_by` field, which Sensu automatically populates with the name of the user who created or last updated each resource.
+- The agent now discovers entity libc type, VM system, VM role, and cloud provider.
+- System type now includes the `float_type` field, which stores the float type the system is using (hard float or soft float).
+- The Bonsai client now logs at the `debug` level rather than the `info` level.
+- The store can now create wrapped resources.
+- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and logs numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
- [Tessen][130] now collects the type of store used for events (`etcd` or `postgres`) and logs numbers of authentication providers, secrets, and secrets providers. Tessen data helps us understand how we can improve Sensu, and all Tessen transmissions are logged locally for complete transparency.
```

**Review by echlebek — APPROVED** [review]

**Comment by hillaryfraley** [comment]
@cwjohnston I think Anna is also calling it "alpha feature" in our next newsletter (I believe she's planning it for tomorrow, so it can still be changed) and our docs call it an alpha feature (https://docs.sensu.io/sensu-go/5.19/sensuctl/reference/#sensuctl-prune)

All of this can be changed, I just wanted to note it somewhere to make sure I get it done if needed.

File: content/sensu-go/5.19/release-notes.md [hunk 16]
```diff
@@ -55,6 +56,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.19.0 release notes
+
+**March 30, 2020** &mdash; The latest release of Sensu Go, version 5.19.0, is now available for download.
+This release is packed with new features, improvements, and fixes, including our first *feature preview* in a release: declarative configuration pruning to help keep your Sensu instance in sync with Infrastructure as Code workflows.
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/release-notes.md]:
```suggestion
This release is packed with new features, improvements, and fixes, including our first alpha feature: declarative configuration pruning to help keep your Sensu instance in sync with Infrastructure as Code workflows.
```

**Review by cwjohnston — APPROVED** [review]
roll with alpha 👍

**Comment by hillaryfraley** [comment]
After discussing with @cwjohnston , we are going with "alpha feature"

**Pull request merged**